mitodl/mit-open
# Add SP as a department

### What are the relevant tickets?
NA

### Description (What does it do?)
This adds SP "Special Programs" as a department

### How can this be tested?
Test with https://github.com/mitodl/course-search-utils/pull/120
https://github.com/mitodl/course-search-utils/pull/120 needs to be merged and released before this PR can be

Set
RUN_DATA_MIGRATIONS=true in your .env file and restart docker
Run `docker-compose run web ./manage.py backpopulate_ocw_data --course-name sp-248-neet-ways-of-thinking-fall-2023 --overwrite`
Go to http://open.odl.local:8062/search/?department=SP and verify that you see that course and and "Special Programs" is in the department facet
Go to http://open.odl.local:8062/c/department/special-programs to verify that the channel exists
